zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/approval/ApproveTask.java

```diff
@@ -159,6 +160,24 @@ private boolean doApproval() throws Exception {
             info().data("user", session.getEmail()).data("collectionId", collection.getDescription().getId())
                     .log("approve task: collection approve task completed successfully");
 
+            // Use only for upload new endpoint
+            if (Configuration.isUploadNewEndpointEnabled()) {
+                // get files here?
+                String fileName = "";
+                for (String string : collectionReader.getReviewed().listUris()) {
+                    if (string.contains("csv") || string.contains("xlsl")) {
```

Comment:
There is a typo here, xlsl should be xlsx and there should also be a check for csdb and xls.